srcpkgs/bubblewrap/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'bubblewrap'
 pkgname=bubblewrap
-version=0.4.1
+version=0.6.1
 revision=2
```

Comment:
revision=1 and see #33372. Is this required for GNOME 42?

Reply:
Also, sorry about removing the `realpath-workaround.patch`, I got mixed up. I updated the patch.